ruma-api-macros/src/api/request.rs

```diff
@@ -517,80 +517,213 @@ impl Request {
             .collect();
 
         quote! {
-                            #[doc = #docs]
```

Comment:
I hadn't noticed that the formatting was completely screwed in this `quote! {}` block. Thanks for fixing it.